test_scripts/API/Transfer_RPC_with_invalid_image/014_OnWayPointChange.lua

```diff
@@ -0,0 +1,80 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0042-transfer-invalid-image-rpc.md
+--
+-- Requirement summary:TBD
+--
+-- Description:
+-- In case:
+-- 1. HMI sends OnWayPointChange with image that is absent on file system
+-- SDL must:
+-- 1. transfer this RPC to mobile app for processing
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/Transfer_RPC_with_invalid_image/common')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local notifParams = {
+  wayPoints =
+  {
+    {
+      coordinate = {
+        latitudeDegrees = -90,
+        longitudeDegrees = -180
+       },
+      locationName = "Ho Chi Minh",
+      addressLines = {"182 Le Dai Hanh"},
+      locationDescription = "Toa nha Flemington",
+      phoneNumber = "1231414",
+      locationImage = {
+        value = common.getPathToFileInStorage("missed_icon.png"),
+        imageType = "DYNAMIC"
+      },
+      searchAddress = {
+        countryName = "aaa",
+        countryCode = "084",
+        postalCode = "test",
+        administrativeArea = "aa",
+        subAdministrativeArea = "a",
+        locality = "a",
+        subLocality = "a",
+        thoroughfare = "a",
+        subThoroughfare = "a"
+      }
+    }
+  }
+}
+
+--[[ Local Functions ]]
+local function subcribleWayPoints()
```

Comment:
typo

Reply:
@jacobkeeler , please find update in bd5458a0785b72d049ed977bb74bb4a0a117cd01